extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+import {log} from '../../../src/log';
+import {loadPromise} from '../../../src/event-helper';
+import {mergeObjects} from '../../../third_party/data-layer-helper/helper';
```

Comment:
agree. theres some extra code there for "non Plain" object which i doubt we need.